communication/mavlink_waypoint_handler.hpp

```diff
@@ -45,13 +46,12 @@
 
 #include "communication/mavlink_message_handler.hpp"
 #include "communication/mavlink_stream.hpp"
-#include "sensing/position_estimation.hpp"
 #include "communication/mavlink_message_handler.hpp"
 #include "communication/state.hpp"
-#include "sensing/qfilter.hpp"
-#include "control/manual_control.hpp"
-#include "control/navigation.hpp"
-#include "control/dubin.hpp"
+#include "mission/navigation.hpp"
+#include "mission/waypoint.hpp"
+
+class Mission_planner;
```

Comment:
done in https://github.com/lis-epfl/MAVRIC_Library/pull/331/commits/f67c5dc06ef3f41407d4fd76f316a790cc0e97de